app/code/Magento/CustomerImportExport/Model/Import/Address.php

```diff
@@ -24,14 +24,12 @@
  */
 class Address extends AbstractCustomer
 {
-    /**#@+
-     * Attribute collection name
+    /**
+     * Customer Address EAV additional attribute collection name
```

Comment:
I think description should be more informative

```suggestion
     * The Customer address attribute collection class name
```